lib/config/package.ts

```diff
@@ -0,0 +1,22 @@
+import { readFileSync } from 'fs';
+
+import { printError } from '../print-error';
+
+import { ExternalConfig } from './types';
+
+export function readFromPackage(): ExternalConfig {
+  let config;
+  try {
+    config = JSON.parse(readFileSync('package.json') as unknown as string);
+  } catch (err) {
+    if ((err as Error).name === 'SyntaxError') {
+      printError('Unable to parse package.json');
+      process.exit(1);
+    }
+    return {};
```

Comment:
Instead of silently falling back to `{}` on some errors, I'd rather always print `err`, like we're doing here: https://github.com/tbroadley/spellchecker-cli/blob/8825f8e462321a296e20dee38b4896caedde8d25/lib/config/file.ts#L25-L26